good-enough-practices-for-scientific-computing.tex

```diff
@@ -1154,9 +1155,8 @@ \section{What We Left Out}\label{sec:omitted}
   this paper.
 
 \item[\textbf{Code Reviews and Pair Programming}] These practices are
-  valuable in projects with multiple contributors, but are hard to
-  adopt in single-author/single-user situations, which includes most
-  of the intended audience for this paper \cite{petre2014}.
+  valuable in projects with multiple people making large software
```

Comment:
We talk a lot about **collaboration** in this paper, but it usually doesn't extend to the modest bits of script or software development in these projects. I reworded this to make it clear that our target audience doesn't have multiple people making contributions to a large complicated codebase.